support.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Get Support
+nav: floating
+---
+
+<div class="section narrow lightest-text" style="margin-bottom: -4rem;">
+	<div class="container">
+		<div class="row">
+			<div class="column two-thirds left">
+				<h1>Get Support</h1>
+				<p>Need help? No problem! Reach out to our community on <a href="//forum.defold.com">forum</a> and <a href="/discord">discord</a>. Additionally, at Defold we offer expert guidance, resources, and assistance to help ensure the long-term success of your game.</p>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="section">
+	<div class="container">
+        <div class="row">
+            <div class="columns six">
+                <h4 style="margin-top: 4rem;">If you are a game developer...</h4>
+                <p>We are committed to helping developers create successful games. That's why we offer the following services:</p>
+                <ul class="checkmark">
+                    <!-- <li>Expert guidance on utilizing the game engine to its full potential</li> -->
+                    <li>Optimization of the game's performance and stability</li>
+                    <li>Creating new editor and engine features</li>
+                    <li>Assistance with technical issues or questions that may arise during the development process </li>
+                    <li>Porting the game to other platforms</li>
+                </ul>        
+                <p>With our support, you can focus on creating an engaging and immersive gaming experience while we take care of the technical details, ensuring that your game is small, performant and stable.</p>
+                <p>Please contact us at <a href="mailto:business@defold.se">business@defold.se</a> for more information on how we can help you with your game development.</p>
+                {% include primary_button.html link="mailto:business@defold.se" text="Contact Us" %}
+            </div>
+            <div class="columns six">
+                <h4 style="margin-top: 4rem;">If you are a vendor or a platform...</h4>
+                <p>We understand the importance of seamless integration with the platforms and services that are used for game development. That's why we offer the following services:</p>
+                <ul class="checkmark">
+                    <li>Professional help on integrating with your platform or product</li>
+                    <li>Creating engine and editor integration with your service</li>
+                    <li>Maintaining and keeping the integration up to date</li>
+                    <li>Providing expert guidance in designing gaming products</li> 
+                </ul>        
+                <p>With our support, you can have the peace of mind that your product is seamlessly integrated into Defold, allowing you to focus on what you do best: creating a great gaming service!</p>
```

Comment:
Capitalize the sentence after a colon if it is a complete sentence:
`best: Creating a great gaming service!`